base/doc/ltnews42.tex

```diff
@@ -148,6 +148,39 @@ \section{Introduction}
 
 \section{News from the Tagged PDF project}
 
+\subsection{Expanding the \cs{DocumentMetadata} command}
+ 
+In 2022 we introduced the \cs{DocumentMetadata} with a twofold purpose:
+It provides a dedicated place for document wide settings and metadata. And it acts as a trigger command to identify documents that want to load new code
+and to use new extended interfaces which are needed for the Tagging Project but are useful also without tagging.
+ 
+At first using \cs{DocumentMetadata} with an empty argument loaded only the PDF management and a new hyperref driver was used. Since november 2024 \cs{DocumentMetadata} changes the default encoding from OT1 but T1 and since june 2025 it changes the default PDF version from 1.7 to 2.0.
+
+Additional code in latex-lab, needed, e.g., for the tagging project, had to be loaded explicitly by using the \texttt{testphase} or the new \texttt{tagging} key in the argument of \cs{DocumentMetadata}. While this allowed to selectively load and test the new code it also makes it difficult for classes and packages to adapt their code for
+the tagging project as it was difficult to test which parts of the latex-lab code were active.
+ 
+In this release we therefore extend \cs{DocumentMetadata} even more: it will now load directly all the code that one get also when using the \texttt{tagging=off} or the \texttt{testphase=latest} key.  
```

Comment:
Better:
... even further: since it now directly loads all the code previously loaded by using either the ...